server/src/test/scala/drt/shared/WalkTimesSpec.scala

```diff
@@ -74,79 +74,79 @@ class WalkTimesSpec extends Specification {
     result === expected
   }
 
-    "When getting a walk time for an arrival" >> {
+  "When getting a walk time for an arrival" >> {
 
-      val gateWalkTimes = Seq(
-        gate1T1,
-        gate2T1,
-        gate1T2,
-      )
+    val gateWalkTimes = Seq(
+      gate1T1,
+      gate2T1,
+      gate1T2,
+    )
 
-      val standWalkTimes = Seq(
+    val standWalkTimes = Seq(
       stand1T1,
       stand2T1,
       stand1T2,
-      )
+    )
 
-      val wt = WalkTimes(gateWalkTimes, standWalkTimes)
+    val wt = WalkTimes(gateWalkTimes, standWalkTimes)
 
-      val walkTimeProvider: (Option[String], Option[String], Terminal) => String = wt.walkTimeForArrival(300000L)
+    val walkTimeProvider: (Option[String], Option[String], Terminal) => String = wt.walkTimeForArrival(300000L)
 
-      "Given a gate and no stand I should get back the gate walk time" >> {
+    "Given a gate and no stand I should get back the gate walk time" >> {
 
-        val result = walkTimeProvider(Option("gate1"), None, T1)
+      val result = walkTimeProvider(Option("gate1"), None, T1)
 
-        result === gate1T1.inMinutes + " walk time"
-      }
+      result === MinuteAsNoun(gate1T1.inMinutes).display + " walk time"
```

Comment:
I see this is how the test was originally written, but I'm not sure it's a good idea to use a function to form the expected value here. It would be clearer if the expected value was explicitly stated, `=== s"${gateT1.inMinutes} minute walk time"`
Same for the next 3 tests

Reply:
yes make sense . I will update that